torch/quantization/fx/utils.py

```diff
@@ -481,3 +482,23 @@ def is_get_tensor_info_node(node: Node) -> bool:
     result: bool = \
         node.op == "call_function" and node.target == getattr and node.args[1] == "shape"  # type: ignore[assignment]
     return result
+
+def maybe_get_next_module(
+    node: Node,
+    modules: Dict[str, nn.Module],
+    is_target_module_type: Callable[[nn.Module], bool]
```

Comment:
can we pass the type directly instead of passing a function which checks for the type?